scripts/test-react.js

```diff
@@ -699,6 +699,23 @@ function runTestSuite(outputJsx, shouldTranspileSource) {
         }
         expect(count).toEqual(8);
       });
+
+      it("Lazy branched elements 2", async () => {
+        let createElement = React.createElement;
+        let count = 0;
+        // $FlowFixMe: intentional for this test
+        React.createElement = (type, config) => {
+          count++;
+          return createElement(type, config);
+        };
+        try {
+          await runTest(directory, "lazy-branched-elements2.js");
+        } finally {
+          // $FlowFixMe: intentional for this test
+          React.createElement = createElement;
+        }
+        expect(count).toEqual(7);
```

Comment:
Please clarify where 7 comes from.

Reply:
The amount of times createElement is called from both compiled and non-compiled. 4 + 3

Reply:
I understand. I'm asking to add it as a comment, similar to what I suggest in https://github.com/facebook/prepack/pull/2107#discussion_r194541733. So that when somebody changes this test, they have an easy reference for how `7` was calculated initially, and thus know whether their new result is correct or not.

Reply:
Yeah, I will do. Sorry, still trying to get my head around why it fails out internal bundle :/